src/Sentry/Transaction.cs

```diff
@@ -360,20 +379,19 @@ public static Transaction FromJson(JsonElement json)
                 Release = release,
                 Distribution = distribution,
                 _request = request,
-                Contexts = contexts ?? new(),
+                Contexts = contexts,
                 _user = user,
                 Environment = environment,
                 Sdk = sdk,
                 _fingerprint = fingerprint,
-                _breadcrumbs = breadcrumbs ?? new(),
-                _extra = extra ?? new(),
-                _tags = (tags ?? new())!,
-                _spans = json
-                    .GetPropertyOrNull("spans")?
-                    .EnumerateArray()
-                    .Select(Span.FromJson)
-                    .ToArray() ?? Array.Empty<Span>()
+                _breadcrumbs = breadcrumbs,
+                _extra = extra,
+                _tags = tags,
+                _measurements = measurements,
+                _spans = spans
```

Comment:
Note, there's some formatting changes and refactoring here also.  The only new part is adding measurements.